lib/dialects/postgres/query.js

```diff
@@ -43,28 +43,18 @@ class Query extends AbstractQuery {
   }
 
   run(sql, parameters) {
-    const connection = this.connection;
-    this.sql = sql;
+    const { connection } = this;
 
     if (!_.isEmpty(this.options.searchPath)) {
-      this.sql = this.sequelize.getQueryInterface().QueryGenerator.setSearchPath(this.options.searchPath) + sql;
+      sql = this.sequelize.getQueryInterface().QueryGenerator.setSearchPath(this.options.searchPath) + sql;
```

Comment:
hmm I think the idea was before to not change the input string by using the "copy" above. I don't know if there is any assumptions made it is immutable or not 🤔 but if the tests are passing, should be ok 
Also, same with the above